UndertaleModTool/Scripts/Builtin Scripts/ShowRoomName.csx

```diff
@@ -1,30 +1,50 @@
+
 // Adds room ID and name display under the debug mode timer display
 
 EnsureDataLoaded();
-
+ScriptMessage("Show room name and ID in debug mode\nby krzys_h, Kneesnap\n");
+Data.Functions.EnsureDefined("room_get_name", Data.Strings); // required for Deltarune
 if (Data?.GeneralInfo?.DisplayName?.Content.ToLower() == "deltarune chapter 1 & 2")
```

Comment:
1) can you externalize the displayname, so that you're not lowering it everytime?
  something like 
  ```cs
  var displayName = Data?.GeneralInfo?.DisplayName?.Content.ToLower();
  if (displayName == "deltarune chapter 1 & 2") { ... }
  ```
  
2) The code for both for *all* blocks is the exact same. why have if conditions at all then?